### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sirsumvent**
I added functionality:
-full 12 frets on the guitar 
     -can display the full 12 frets or smaller depending on user input
-any combination of notes to be displayed

### CODE REVIEW

**Review by denimad — CHANGES_REQUESTED** [review]

File: GUITAR.py [hunk 1]
```diff
@@ -1,49 +1,145 @@
...
-    print()
+print('Once you are done, type "stop"')
+print()
+print()
+
+while True:
+    looking = input('What note are you looking for?').casefold()
+    # looking for a particular note:
+    if looking == 'c':
+        looking = '4, 3, 1, 1, 2, 5, 0, 8, 3, 10, 5, 8'  # coordinates (x, y) values
```
[the remark this answers is not in the document]
Comment (sirsumvent) [reply GUITAR.py:24]: string has been corrected to a more proper data structure: a list.

File: GUITAR.py [hunk 2]
```diff
@@ -21,112 +21,112 @@
...
-        x1, x2, x3, x4, x5, x6, x7, x8 = looking[0], looking[6], looking[12], looking[18], looking[24], looking[30], looking[37], looking[44]  # index for x value
-        y1, y2, y3, y4, y5, y6, y7, y8 = looking[3], looking[9], looking[15], looking[21], looking[27], looking[33:35], looking[40:42], looking[47]
+        looking = [0, 0, 1, 5, 5, 0, 3, 2, 4, 7, 0, 12, 5, 12, 2, 9]
+        x1, x2, x3, x4, x5, x6, x7, x8 = looking[0], looking[2], looking[4], looking[6], looking[8], looking[10], looking[12], looking[14]  
+        y1, y2, y3, y4, y5, y6, y7, y8 = looking[1], looking[3], looking[5], looking[7], looking[9], looking[11], looking[13], looking[15]
         # find x,y values and replace with ' '
         neck[int(x1)][int(y1)], neck[int(x2)][int(y2)], \
         neck[int(x3)][int(y3)], neck[int(x4)][int(y4)], \
         neck[int(x5)][int(y5)], neck[int(x6)][int(y6)], \
         neck[int(x7)][int(y7)], neck[int(x8)][int(y8)] = '|-e-', '-E-', '|-E-', '-E-', '-E-', '-E-', '-E-', '-E-'
```
Comment (denimad) [inline GUITAR.py:69]: little 'e'? this is why repeating to many time variables, such as a string, isn't good, it's prone to errors. Why not creating a constant and referring to it every time you want to write '-E-'. 
Also, Figuring out a way to remove the "special cases" when you need to add the '|' symbol will make the code much cleaner.

**Review by denimad — CHANGES_REQUESTED** [review]

**Review by denimad — CHANGES_REQUESTED** [review]
second iteration on patch-1 PR.

File: GUITAR.py (hunk 2 above)
Comment (sirsumvent) [reply GUITAR.py:69]: The little 'e' is to denote that it is the 6th string, the thinnest string. I believe it's more a stylistic choice than anything. I'll keep it in.

**Pull request merged**